thorlcr/activities/indexwrite/thindexwriteslave.cpp

```diff
@@ -629,11 +617,11 @@ class IndexWriteSlaveActivity : public ProcessSlaveActivity, public ILookAheadSt
             mb.append(inactiveStats.getStatisticValue(StNumLeafCacheAdds));
             mb.append(inactiveStats.getStatisticValue(StNumBlobCacheAdds));
             mb.append(inactiveStats.getStatisticValue(StNumNodeCacheAdds));
-            mb.append(offsetBranches);
+            mb.append(inactiveStats.getStatisticValue(StNumOffsetBranches));
             mb.append(uncompressedSize);
             mb.append(originalBlobSize);
-            mb.append(branchMemorySize);
-            mb.append(leafMemorySize);
+            mb.append(inactiveStats.getStatisticValue(StSizeBranchMemory));
+            mb.append(inactiveStats.getStatisticValue(StSizeLeafMemory));
```

Comment:
these stats are now already being serialized to the manager (via regular stats. serialization),
so I think these lines should be deleted, and the corresponding manual deserialization in the manager activity code, and the manager activity should pick up these values from CMasterActivity::statsCollection..

e.g. in IndexWriteActivityMaster::done:
```
            props.setPropInt64("@numLeafNodes", statsCollection.getStatisticSum(StNumLeafCacheAdds));
```

Reply:
not urgent, so can be left to a subsequent JIRA/PR if you prefer?
Should be a relatively easy change and avoid future confusion re. 2 places these stats. are serialized/deserialized.